src/pandorasat/visibledetector.py

```diff
@@ -87,25 +88,31 @@ def _dispersion_df(self):
 
     @property
     def dark(self):
+        """Dark Noise"""
         return 1 * u.electron / u.second
 
     @property
     def read_noise(self):
+        """Read Noise"""
         return 1.5 * u.electron
 
     @property
     def bias(self):
+        """Detector bias""""
```

Comment:
```suggestion
        """Detector bias"""
```
One too many speech marks is tripping the test!